LetsDoStuff.Domain/LdsContext.cs

```diff
@@ -7,10 +7,27 @@ public class LdsContext : DbContext
     {
         public DbSet<User> Users { get; set; }
 
+        public DbSet<Activity> Activities { get; set; }
+
+        public DbSet<Tag> Tags { get; set; }
+
         public LdsContext(DbContextOptions<LdsContext> options)
             : base(options)
         {
             Database.EnsureCreated();
         }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=relationsdb;Trusted_Connection=True;");
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Activity>()
+                    .HasMany(a => a.Tags)
+                    .WithMany(t => t.Activities)
+                    .UsingEntity(j => j.ToTable("Enrollments"));
```

Comment:
You could use a many-to-many relationship. If you update to .net 5. See the https://metanit.com/sharp/entityframeworkcore/3.6.php